### ISSUE TITLE
jsonschema fails on Windows due to missing dependency / resource

### ISSUE THREAD

**Issue by duk3luk3** [FAForever/client#984]
Looks like a missing file that needs to be copied into the installer.

```
Traceback (most recent call last):
  File "C:\Python36\lib\site-packages\cx_Freeze\initscripts\__startup__.py", line 14, in run
  File "C:\Python36\lib\site-packages\cx_Freeze\initscripts\Console.py", line 26, in run
  File "src/__main__.py", line 140, in <module>
  File "src/__main__.py", line 102, in runFAF
  File "src\client\_clientwindow.py", line 519, in setup
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 656, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 626, in _load_backward_compatible
  File "src\replays\__init__.py", line 5, in <module>
OSError: [Errno 0] Error: 'jsonschema\\schemas\\draft3.json'

Runtime info:

FAF Username: pre-login
FAF Version: 0.16.0+171-g39802928.254
FAF Environment: production
FAF Directory: C:\ProgramData\FAForever
FA Path: : Unknown
Home Directory: C:/Users/Lukas/Documents
Platform: Windows-7-6.1.7601-SP1
Uname: uname_result(system='Windows', node='Lukas-PC', release='7', version='6.1.7601', machine='AMD64', processor='Intel64 Family 6 Model 61 Stepping 2, GenuineIntel')
```

**Comment by duk3luk3** [comment]
Reported to upstream, where the bug is maybe (?)

**Comment by Wesmania** [comment]
It might be an issue with cx_Freeze: https://bitbucket.org/anthony_tuininga/cx_freeze/issues/93/jsonschema-package-required-for-ipython

If it is, then we'll need to unfreeze cx_Freeze dependency (and fix the issue with cx_Freeze putting modules in 'lib' subdirectory, probably by renaming our own 'lib' to 'bin' for example).

**Comment by Wesmania** [comment]
Fixed by #985.
